wca-regulations.md

```diff
@@ -521,5 +521,8 @@ Organization teams may adopt optional regulations to facilitate the administrati
 - Z1) The organization team may require competitors to submit puzzles during registration.
 - Z2) The organization team may limit the number of events per competitor.
 - Z3) The organization team may select competitors who directly qualify for certain rounds of certain events based on the results of specific previous competitions.
-- Z4) The organization team may limit the number of competitors per event or per competition, on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+- Z4) The organization team may limit the number of competitors per event or per competition, on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date. If the number of competitors per event or per competition is limited, the following regulations apply:
+    - Z4a) If the limit for a competition or an event is reached before the competition, the organization team must not accept on-site registrations for the competition or the event.
+    - Z4b) Competitor limits should be announced together with the competition. Any changes to competitor limits are at the discretion of the WCA Board.
+    - Z4c) If there is more than one registration period, the organization team must announce each additional registration period at least 48 hours before it starts.
```

Comment:
This implies that you can announce on-site registrations up to 48 hours in advance, right?

Reply:
Yes, right.

Reply:
And how should that announcement happen? I think that the vast majority of people won't see it regardless of the method used.

I think that the Chinese system is clear and fair, so we shouldn't forbid it, but I don't feel like leaving the door open to re-opening registrations with an announcement 48 hours before is the right thing to do. My suggestion is to change Z4c) to something like:

> If there is more than one registration period, the organization team must clearly state when each additional registration period will happen on the competition website before the competition is announced.

Reply:
I don't have a strong opinion between the two options.
I think the main issue(s) with reopening registrations were stated by Laura in the PR description, and either way I feel like it seriously prevent abuse.

Reply:
Yes, I considered something like this but I think it's too strict. There can be valid reasons to reopen the registration although it wasn't planned before. It's sometimes difficult to predict registrations/cancellations, so this is not necessarily bad planning.

Additionally, forbidding any additional registration periods would also cause problems when the competitor limit is raised after the registration was closed.

Reply:
Ok, I'm fine with leaving that door open.

Now that I read Z4b) again, shouldn't it be "competitors limits" instead of "competitor limits"?

Reply:
Why do you think so?
It's also "Competitor Limit" on the competition pages.

Reply:
I don't get it: you can advance on-site registration 48 hours in advance, but then Z4a says it is not allowed. Isn't it a contradiction here?

Reply:
Can you explain where you see the problem? Where is the contradiction?

Z4a does only forbid on-site registrations when the competitor limit was reached.

Reply:
Ah, I see. And if the limit was once reached, but then several competitors dropped out, the limit is not considered as reached anymore, right? (The Chinese system)
Sounds good then.